docs/observability/README.md

```diff
@@ -0,0 +1,81 @@
+# Observability
+This README describes observability characteristics of the Fleet Manager
+
+## Metrics
+The file [metrics.go](../../pkg/metrics/metrics.go) creates Prometheus metrics
+of differing types. These are the metrics which are then reported and visualised
+in each Grafana dashboard.
+See [here](https://prometheus.io/docs/concepts/metric_types/) for more info
+about Prometheus metric types.
+
+These metrics are grouped by metric subject: data plane clusters,
+service ('kafka') and reconcilers.
+
+There is also a section of metrics regarding the Observatorium service. See [kas-fleet-manager metrics configmap](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/observability/grafana/grafana-dashboard-kas-fleet-manager-metrics.configmap.yaml#L5460-7247)
+for an example of a Grafana dashboard reporting these Observatorium-related metrics.
+
+The file [metrics_middleware.go](../../pkg/handlers/metrics_middleware.go)
+creates metrics concerned with incoming requests to the Fleet Manager API. This
+file contains information about how these metrics are written and reported.
+
+## Grafana Dashboards
+See the [Getting Started](https://grafana.com/docs/grafana/latest/getting-started/?pg=docs)
+section of the Grafana website for general information about Grafana and its
+uses.
+
+See the [JSON Model](https://grafana.com/docs/grafana/latest/dashboards/json-model/?pg=docs)
+section for more info about Grafana Dashboard JSON models
+
+## Further information
+See [SLOs README](../slos/README.md) for more informtion about metrics and
+their use in measuring SLIs.
+
+> NOTE this document contains references to Red Hat internal components
+
+See [here](https://gitlab.cee.redhat.com/service/app-interface#add-a-grafana-dashboard)
+for information about adding Grafana dashboards in App-Sre
+
+## Observatorium
+
+The service that the Fleet Manager manages
+can send metrics to a [Observatorium](https://github.com/observatorium/observatorium)
+instance from the data plane. Fleet Manager is also able to interact directly
+with Observatorium to retrieve the metrics sent by the managed
+service from the data plane.
+
+### Configuring Observatorium
+
+To configure a new managed service to use Observatorium a Red Hat managed
+Observatorium service can be used. For that, a new `Observatorium Tenant` has
+to be created and configured in that Red Hat managed Observatorium service. That
+task is done by the Red Hat Observability team. To do so there's
+an Onboarding process. See the[Onboarding a Tenant into Red Hat’s Observatorium Instance](https://docs.google.com/document/d/1pjM9RRvij-IgwqQMt5q798B_4k4A9Y16uT2oV9sxN3g) document on how to do it.
```

Comment:
Is this relevant to an already deployed service?

Reply:
This repository doesn't assume RHOSAK as the only audience.
Not all users potential users/developers of KFM are RHOSAK service

Reply:
The intent is then to have this section so that other services can know how to request a tenant to Observatorium?

Reply:
Not other services, but other users that want to deploy their own KFM

Reply:
I think I get the intent of what you are describing now. I'd suggest to reword the section in a way that it clearly reflects that intent.
For example, 

`"To configure a new managed service to use Observatorium a Red Hat managed
Observatorium service can be used. For that, a new `Observatorium Tenant` has
to be created and configured in that Red Hat managed Observatorium service...."` 

can be rephrased to (just a suggestion) 

`"To configure kas-fleet-manager to use Red Hat managed Observatorium, a tenant & service account credentals (separate from RHOSAK's Stage&Prod ones) have to be created in Red Hat managed Observatorium. That
task is done by the Red Hat Observability team via an onboarding process explained in the[Onboarding a Tenant into Red Hat’s Observatorium Instance](https://docs.google.com/document/d/1pjM9RRvij-IgwqQMt5q798B_4k4A9Y16uT2oV9sxN3g) document."`

Reply:
That makes sense :+1: . Updated